src/common/queries/api-webhooks.ts

```diff
@@ -11,30 +11,12 @@
 import { endpoint } from '$app/common/helpers';
 import { request } from '$app/common/helpers/request';
 import { useQuery } from 'react-query';
-import { Params } from './common/params.interface';
 import { GenericSingleResourceResponse } from '$app/common/interfaces/generic-api-response';
 import { ApiWebhook } from '$app/common/interfaces/api-webhook';
 import { toast } from '$app/common/helpers/toast/toast';
 import { useAdmin } from '$app/common/hooks/permissions/useHasPermission';
 import { $refetch } from '../hooks/useRefetch';
 
-export function useApiWebhooksQuery(params: Params) {
```

Comment:
@beganovich The hook is not used in any part of the app's logic, so I removed it.